doc/guides/bluetooth/autopts/autopts-linux.rst

```diff
@@ -233,6 +233,11 @@ Remember to install drivers from installation directory
    :width: 850
    :align: center
 
+.. note::
+
+    In latest versions of PTS there is no Bluetooth Protocol Viewer included.
```

Comment:
Perhaps we need to specify which version it was no longer included, e.g.

```
In PTS versions newer than 5.53 there is no Bluetooth Protocol Viewer included.
```

Where `5.53` of course should be the right version